lib/cppcheck.rb

```diff
@@ -9,11 +9,20 @@ def setup
     @file_wrapper = @ceedling[:file_wrapper]
     @loginator = @ceedling[:loginator]
     @setupinator = @ceedling[:setupinator]
+    @system_wrapper = @ceedling[:system_wrapper]
     @tool_executor = @ceedling[:tool_executor]
     @tool_validator = @ceedling[:tool_validator]
-
+    
     @config = @setupinator.config_hash[CPPCHECK_SYM]
     
+    @config.each do |key,value|
```

Comment:
`value` is not used inside the block, it should be removed.

Reply:
or may you want to use it for the match and as the argument for `module_eval()`?

Reply:
I did what you proposed, and the Array is also handled.